client/blocks/reader-full-post/_content.scss

```diff
@@ -442,3 +442,70 @@
 	padding-left: 16px;
 	padding-right: 16px;
 }
+
+// Writing Prompts Block Styles
+.reader-full-post__story-content .jetpack-blogging-prompt {
+	border: 1px solid var(--color-neutral-5);
```

Comment:
I went ahead and just used a hex value here to save you the hassle. Updated in: https://github.com/Automattic/wp-calypso/pull/76092/commits/3b86368cd9cf3ab0f8ca6dc12358aaa51615fedf

Reply:
I think it would be better to use `--color-neutral-5` in this case. Can you revert that change?

We already have other `--color-neutral` usages currently and we do change the actual hex color being used depending on light vs dark mode.

For instance, in this case, we would use a very light gray on the white background (light mode) and a darker gray on the dark background (dark mode), so the border is always subtle. Having the color hardcoded to `#dcdcde` makes the border too contrasting, here's an example of how it is looking on Android right now:
<img src="https://user-images.githubusercontent.com/5091503/234123339-04a76d01-fd4c-4612-9891-2619086b5ed7.png" height="600">

I was already working on implementing the `--color-neutral-5` today, and this is what it looks like:
<img src="https://user-images.githubusercontent.com/5091503/234123641-986641e7-fe3e-4980-95c0-bd538968a519.png" height="600">

Reply:
I reverted it in https://github.com/Automattic/wp-calypso/pull/76092/commits/e4c77c6d576840ed74946a4182a50560af69fa30.